modules/core/server/services/influx.server.service.js

```diff
@@ -88,5 +111,7 @@ exports.writeMeasurement = function(measurementName, fields, tags, callback) {
       }
     ]);
 
+    callback();
```

Comment:
Is this causing tests to hang up? https://travis-ci.org/Trustroots/trustroots/builds/179439231